modules/nw-sriov-topology-manager.adoc

```diff
@@ -10,7 +10,7 @@ You can create a NUMA aligned SR-IOV pod by restricting SR-IOV and the CPU resou
 .Prerequisites
 
 * Install the OpenShift CLI (`oc`).
-* Enable a LatencySensitive profile and configure the CPU Manager policy to `static`.
+* Configure the CPU Manager policy to `static`.
```

Comment:
I think it missed some steps before, so to guarantee that the container will use SR-IOV device and CPUs from the same NUMA node you should.
1. Configure the CPU Manager policy to `static` and it can be a good idea to specify the reserved CPUs flag as well.
2. Configure the topology manager policy to singleNUMAnode

Reply:
@cynepco3hahue - I have made the suggested updates. Kindly review.